From 20a247b4a7a39ed7dff07f51f081e6d73cc8b598 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <bochecha@daitauha.fr>
Date: Wed, 22 Aug 2018 14:09:49 +0200
Subject: [PATCH 27/27] meson: Run the install script with Python 3

The script was being run directly as an executable, and it has a
Python 2 shebang.

Reviewed-by: Eric Engestrom <eric.engestrom@intel.com>
Reviewed-by: Emil Velikov <emil.velikov@collabora.com>
---
 src/gallium/targets/dri/meson.build   | 1 +
 src/gallium/targets/va/meson.build    | 1 +
 src/gallium/targets/vdpau/meson.build | 1 +
 src/gallium/targets/xvmc/meson.build  | 1 +
 src/mesa/drivers/dri/meson.build      | 1 +
 5 files changed, 5 insertions(+)

diff --git a/src/gallium/targets/dri/meson.build b/src/gallium/targets/dri/meson.build
index e3202c9967..51940a2f35 100644
--- a/src/gallium/targets/dri/meson.build
+++ b/src/gallium/targets/dri/meson.build
@@ -84,6 +84,7 @@ foreach d : [[with_gallium_pl111, 'pl111_dri.so'],
 endforeach
 
 meson.add_install_script(
+  prog_python.path(),
   join_paths(meson.source_root(), 'bin/install_megadrivers.py'),
   libgallium_dri.full_path(),
   dri_drivers_path,
diff --git a/src/gallium/targets/va/meson.build b/src/gallium/targets/va/meson.build
index 57d267e5b6..ded689b464 100644
--- a/src/gallium/targets/va/meson.build
+++ b/src/gallium/targets/va/meson.build
@@ -62,6 +62,7 @@ foreach d : [[with_gallium_r600, 'r600'],
 endforeach
 
 meson.add_install_script(
+  prog_python.path(),
   join_paths(meson.source_root(), 'bin/install_megadrivers.py'),
   libva_gallium.full_path(),
   va_drivers_path,
diff --git a/src/gallium/targets/vdpau/meson.build b/src/gallium/targets/vdpau/meson.build
index 93969d8399..005cf64249 100644
--- a/src/gallium/targets/vdpau/meson.build
+++ b/src/gallium/targets/vdpau/meson.build
@@ -65,6 +65,7 @@ foreach d : [[with_gallium_r300, 'r300'],
 endforeach
 
 meson.add_install_script(
+  prog_python.path(),
   join_paths(meson.source_root(), 'bin/install_megadrivers.py'),
   libvdpau_gallium.full_path(),
   vdpau_drivers_path,
diff --git a/src/gallium/targets/xvmc/meson.build b/src/gallium/targets/xvmc/meson.build
index 2bf5ea2309..0af5b6477c 100644
--- a/src/gallium/targets/xvmc/meson.build
+++ b/src/gallium/targets/xvmc/meson.build
@@ -56,6 +56,7 @@ foreach d : [[with_gallium_r600, 'r600'], [with_gallium_nouveau, 'nouveau']]
 endforeach
 
 meson.add_install_script(
+  prog_python.path(),
   join_paths(meson.source_root(), 'bin/install_megadrivers.py'),
   libxvmc_gallium.full_path(),
   xvmc_drivers_path,
diff --git a/src/mesa/drivers/dri/meson.build b/src/mesa/drivers/dri/meson.build
index 943727b662..d98c823f5f 100644
--- a/src/mesa/drivers/dri/meson.build
+++ b/src/mesa/drivers/dri/meson.build
@@ -57,6 +57,7 @@ if dri_drivers != []
   )
 
   meson.add_install_script(
+    prog_python.path(),
     join_paths(meson.source_root(), 'bin/install_megadrivers.py'),
     libmesa_dri_drivers.full_path(),
     dri_drivers_path,
-- 
2.18.1

